test/java/src/test/resources/test4.xml

```diff
@@ -0,0 +1 @@
+<UserModel><AddressList><Address><AddressID>1</AddressID><Address1>3000 Flinders Street</Address1><City>Melbourne</City><Country>AU</Country><State>VIC</State><ZipCode>3001</ZipCode></Address><Address><AddressID>3</AddressID><Address1>ANZ Building</Address1><Address2>833 Collins Street</Address2><City>Melbourne</City><Country>AU</Country><State>VIC</State><ZipCode>3000</ZipCode></Address></AddressList><UserAddressRelList><UserAddressRel><AddressID>1</AddressID><UserID>2</UserID><AddressType>Home</AddressType></UserAddressRel><UserAddressRel><AddressID>3</AddressID><UserID>2</UserID><AddressType>Office</AddressType></UserAddressRel></UserAddressRelList><UsersList><Users><UserID>2</UserID><PrimaryAddressID>1</PrimaryAddressID><FirstName>John</FirstName><LastName>Smith</LastName></Users></UsersList></UserModel>
```

Comment:
Could you please format this with newlines and indents?

Reply:
Unfortunately no. ATM Deserialiser does not like formatted input.

Reply:
I see, not to worry then.

Reply:
Raise an issue to fix this.

Reply:
Yup, please raise issue and refer to it here.

Reply:
Raised issue #81